src/Avalonia.Native/WindowImpl.cs

```diff
@@ -220,5 +220,8 @@ public void SetEnabled(bool enable)
         {
             _native.SetEnabled(enable.AsComBool());
         }
+        
+        // TODO: Implement
+        public bool IsEnabled => throw new NotImplementedException();
```

Comment:
Does this need to be implemented in this PR? I think throwing here will currently break DnD on OSX, correct?

Reply:
yes, i've no experience going low level with OSX so i'd rather someone else wrote this part

Reply:
Oh yeah sorry that's in the description. Need to learn to read.